arch/TriCore/TriCoreInstPrinter.c

```diff
@@ -416,7 +416,7 @@ static void printDisp8Imm(MCInst *MI, int OpNum, SStream *O)
 			break;
 		}
 
-		SS_print_sign_hex(O, disp);
+		SS_print_hex(O, disp);
```

Comment:
Better use `printUInt32()` or `printInt32()` here. They are implemented in `SStream.c`